internal/controllers/machinedeployment/machinedeployment_status_test.go

```diff
@@ -414,10 +421,11 @@ func Test_setScalingUpCondition(t *testing.T) {
 				Status: metav1.ConditionFalse,
 				Reason: clusterv1.MachineDeploymentNotScalingUpReason,
 			},
+			expectedPhase: clusterv1.MachineDeploymentPhaseRunning,
 		},
 		{
 			name:              "not scaling up with machines",
-			machineDeployment: deletingMachineDeploymentWith3Replicas,
+			machineDeployment: scalingUpMachineDeploymentWith3Replicas,
```

Comment:
scalingUpMachineDeploymentWith3Replicas + "not scaling up with machines" is a bit awkward